src/sections/layer_and_mask_information_section/mod.rs

```diff
@@ -51,9 +53,7 @@ pub mod layer;
 /// | Variable | (Photoshop 4.0 and later) <br> Series of tagged blocks containing various types of data. See See Additional Layer Information for the list of the types of data that can be included here. |
 #[derive(Debug)]
 pub struct LayerAndMaskInformationSection {
-    pub(in crate) layers: Vec<PsdLayer>,
-    /// A map of layer name to index within our layers vector
-    pub(in crate) layer_names: HashMap<String, usize>,
+    pub(crate) group: PsdLayerGroup,
```

Comment:
I'm finding this to be confusing

Wouldn't the layer and mask information section have names and groups?

Maybe something like:

```rust
pub struct LayerAndMaskInformationSection {
    layers: Vec<PsdLayer>,
    layer_indices: HashMap<String, usize>,
    groups: HashMap<String, Range<usize>>
}
```